### PR TITLE
Add StarmapRouteSearch

### PR DESCRIPTION
**Pull request by henry232323**
Adds the Starmap Route searching endpoint which takes in the names of two objects and calculates the fastest route between them using this endpoint: https://robertsspaceindustries.com/api/starmap/routes/find

### CODE REVIEW

File: rsi_scraper/starmap.py [hunk 1]
```diff
@@ -224,6 +226,48 @@ def execute(self):
         return res
 
 
+# {"departure":"STANTON.LZS.LORVILLE","destination":"STANTON.LZS.AREA18","ship_size":"L"}
```
Comment (Dymerz) [inline rsi_scraper/starmap.py]: Need to be removed

**Review by Dymerz — CHANGES_REQUESTED** [review]
Thank you! The code looks good, I think you could pass the `ship_size` as a parameters too.
Also I'm going add the unit test 🙂

File: rsi_scraper/starmap.py [hunk 2]
```diff
@@ -224,6 +226,48 @@ def execute(self):
...
+class StarmapRouteSearch(ICommand):
+    """StarmapRouteSearch
+    """
+
+    __url_find = "https://robertsspaceindustries.com/api/starmap/routes/find"
+
+    def __init__(self, _from: str, to: str):
+        """
+        Args:
+            search (str): The search.
```
Comment (Dymerz) [inline rsi_scraper/starmap.py]: Update the documentation.

File: rsi_scraper/starmap.py [hunk 3]
```diff
@@ -224,6 +226,48 @@ def execute(self):
...
 
+# {"departure":"STANTON.LZS.LORVILLE","destination":"STANTON.LZS.AREA18","ship_size":"L"}
+
+class StarmapRouteSearch(ICommand):
+    """StarmapRouteSearch
+    """
+
+    __url_find = "https://robertsspaceindustries.com/api/starmap/routes/find"
+
+    def __init__(self, _from: str, to: str):
```
Comment (Dymerz) [inline rsi_scraper/starmap.py]: Pass `ship_size` parameter.

**Review by Dymerz — CHANGES_REQUESTED** [review]

**Comment by Dymerz** [comment]
@henry232323 Can you allow edits from maintainers in this PR so I can commit to this one? ([more about](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))

**Comment by henry232323** [comment]
Should be enabled

**Review by Dymerz — DISMISSED** [review]

**Review by Dymerz — APPROVED** [review]

**Comment by Dymerz** [comment]
It all good, Thanks!

**Pull request merged**